<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="RegressionSuite" duration-ms="4926" started-at="2022-02-13T02:57:56Z" finished-at="2022-02-13T02:58:01Z">
    <groups>
    </groups>
    <test name="MyAccount" duration-ms="951" started-at="2022-02-13T02:57:56Z" finished-at="2022-02-13T02:57:57Z">
      <class name="yourStoreTestCases.MyAccountTestCase">
        <test-method status="PASS" signature="myAccount()[pri:0, instance:yourStoreTestCases.MyAccountTestCase@10bdf5e5]" name="myAccount" duration-ms="618" started-at="2022-02-12T21:57:56Z" finished-at="2022-02-12T21:57:57Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- myAccount -->
      </class> <!-- yourStoreTestCases.MyAccountTestCase -->
    </test> <!-- MyAccount -->
    <test name="Register" duration-ms="2274" started-at="2022-02-13T02:57:57Z" finished-at="2022-02-13T02:58:00Z">
      <class name="yourStoreTestCases.RegisterTestCase">
        <test-method status="FAIL" signature="Register()[pri:0, instance:yourStoreTestCases.RegisterTestCase@3108bc]" name="Register" duration-ms="2014" started-at="2022-02-12T21:57:57Z" finished-at="2022-02-12T21:57:59Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Your Account Has Been Created!] but found [Register Account]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Your Account Has Been Created!] but found [Register Account]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at yourStoreTestCases.RegisterTestCase.Register(RegisterTestCase.java:42)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Register -->
      </class> <!-- yourStoreTestCases.RegisterTestCase -->
    </test> <!-- Register -->
    <test name="Login" duration-ms="1677" started-at="2022-02-13T02:58:00Z" finished-at="2022-02-13T02:58:01Z">
      <class name="yourStoreTestCases.LoginTest">
        <test-method status="PASS" signature="init()[pri:0, instance:yourStoreTestCases.LoginTest@3e9b1010]" name="init" is-config="true" duration-ms="6126" started-at="2022-02-12T21:57:50Z" finished-at="2022-02-12T21:57:56Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- init -->
        <test-method status="PASS" signature="login()[pri:0, instance:yourStoreTestCases.LoginTest@3e9b1010]" name="login" duration-ms="1641" started-at="2022-02-12T21:58:00Z" finished-at="2022-02-12T21:58:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- login -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:yourStoreTestCases.LoginTest@3e9b1010]" name="tearDown" is-config="true" duration-ms="102" started-at="2022-02-12T21:58:01Z" finished-at="2022-02-12T21:58:01Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- yourStoreTestCases.LoginTest -->
    </test> <!-- Login -->
  </suite> <!-- RegressionSuite -->
</testng-results>
